permissions/plugin-kotlin-v1-stdlib-jdk7.yml

```diff
@@ -0,0 +1,6 @@
+---
+name: "kotlin-v1-stdlib-jdk7"
```

Comment:
Not sure it's actually required anymore. 2.60+ is probably enough for new plugins

Reply:
Your properly right lets avoid it 👍